todo/dto/deferred_details_dto.py

```
@@ -0,0 +1,9 @@
+from pydantic import BaseModel
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)